indico/htdocs/js/indico/modules/events/cloning.js

```diff
@@ -61,13 +61,15 @@
                 success: function(data) {
                     if (data.success) {
                         var $countNumber = $eventCount.find('.count');
+                        var $info = $eventCount.find('.info');
                         $countNumber.text(data.count);
                         $cloneErrors.hide();
                         $eventCount.show();
 
                         $eventList.toggle(!!data.count);
                         $cloneButton.prop('disabled', !data.count);
                         $eventCount.data('event-dates', data.dates);
+                        data.additional_info ? $info.text(data.additional_info) : $info.text('');
```

Comment:
`?:` should not be used as a replacement for if..else but only when actually using the return value of the expressions.

`$info.text(data.additional_info ? data.additional_info : '');` would be OK, but this can be simplified to

    $info.text(data.additional_info || '');